tests/service/FsUnit.fs

```diff
@@ -35,32 +35,4 @@ let shouldPairwiseEqual (x: seq<_>) (y: seq<_>) =
     while ex.MoveNext() do countx <- countx + 1
     while ey.MoveNext() do county <- county + 1
     if countx <> county then        
-        Assert.Fail("Collections are of unequal lengths, expected length {0}, actual length is {1}.", countx, county)
-
-let notEqual x = NotConstraint(EqualConstraint(x))
```

Comment:
Most of these were not used at all. A few were used in a couple of places, I replaced them with equivalent (e.g. used `Assert.True` instead of `True` constraint).